.goreleaser.docker.yml

```diff
@@ -48,6 +48,24 @@ dockers:
     use: "buildx"
     build_flag_templates:
       - "--platform=linux/arm64"
+  - image_templates:
+      - &amd_image "authzed/zed:{{ .Version }}-amd64"
```

Comment:
isn't this redefining the anchor for `amd_image`? I think this will fail if we try to run it.

in spicedb we did: https://github.com/authzed/spicedb/blob/main/.goreleaser.yml